app/helpers/users_helper.rb

```diff
@@ -26,7 +26,11 @@ def edit_user_path_by_user_type(user)
   end
 
   def is_org_admin?
-    current_user.role == 'organisation_admin'
+    current_user.organisation_admin?
+  end
+
+  def is_super_org_admin?
+    current_user.super_organisation_admin?
```

Comment:
This method is used in some of the views in the previous commit so we probably want to add it there, not here.